docs/deployment/troubleshoot_deployment.md

```diff
@@ -1,33 +1,45 @@
 # Troubleshooting Cumulus Deployment
 
-This document provides 'notes' on frequently encountered deployment issues.
+This document provides 'notes' on frequently encountered deployment issues. he issues reported are organized by relevant subsection.
 
-The issues reported are organized by relevant subsection, and *may* be out of date given the current rate of development.   Use at your own risk.
+### Configuring the Cumulus Stack
+#### VPC
 
-## Deploy Cumulus
-### Installation
 Issues:
 
-- If you have a `PEM_read_bio` error when bootstrapping, SSL certificates may be to blame
+  - If redeploying an existing configuration you may already have at least 1 vpc associated with your existing deployment, but its subnets can be transitory in nature depending on what kind of load balancing and/or docker activities are taking place at a given time.  You should  identify at least one persistent subnet to use as a subnet ID (you may only specify one) for use.    If this is needed, navigate to  [AWS EC2 > Auto Scaling Groups](https://console.aws.amazon.com/ec2/autoscaling/home) and note the "Availability Zone" (e.g., us-east-1a). Next, visit [AWS VPC](https://console.aws.amazon.com/vpc/home) and click on "Subnets". Copy the 'VPC' value into 'vpcId' and the appropriate 'Subnet ID' value, based on the Availability Zone value you just saw on the Auto Scaling Groups page, into 'subnets'. If you have no vpc and/or subnets, do not include the vpc section in your new configuration.
```

Comment:
the link for `[AWS EC2 > Auto Scaling Groups](https://console.aws.amazon.com/ec2/autoscaling/home)` doesn't take me to the correct place.  It takes me to launch configurations and I didn't see anything with an availability zone on it.
would this link work? `[AWS EC2 > Auto Scaling Groups](https://console.aws.amazon.com/ec2/autoscaling/home?region=us-east-1#AutoScalingGroups:view=details)`

Reply:
Can we put an example of the configuration in here.
```
  vpc:
    vpcId: vpc-34776d4d
    subnets:
      - subnet-e5f0a8e9
```

Reply:
Maybe say you need to find the vpcId and subnet for your availability zone up front.